MachObfuscator/Options/Options.swift

```diff
@@ -131,6 +143,13 @@ extension Options {
                 eraseSymtab = false
             case OptLongCases.swiftReflection.rawValue:
                 swiftReflectionObfuscation = true
+            case OptLongCases.objcBlacklistSelector.rawValue:
+                objcOptions.selectorsBlacklist += String(cString: optarg).split(separator: ",").map { String($0) }
+            case OptLongCases.objcBlacklistSelectorRegex.rawValue:
+                guard let regex = try? NSRegularExpression(pattern: String(cString: optarg), options: []) else {
```

Comment:
What do you think about using `do try catch` instead of `if let try?`? Displaying detailed error message could improve the user experience a bit.